pages/api/auth/[...nextauth].js

```diff
@@ -2,6 +2,7 @@ import NextAuth from 'next-auth'
 import axios from 'axios'
 // TODO(alishaevn): use the api value from https://github.com/assaydepot/rx/issues/21497 in the next phase
 import { EXPIRATION_DURATION, getWebhookConfig, createWebhookConfig } from '../../../utils'
+const authorizationDomain = process.env.NEXT_PUBLIC_CLIENT_DOMAIN || 'scientist.com'
```

Comment:
added the fallback so that the webstore still works when this work is contributed back.